src/tcgwars/logic/impl/gen3/CrystalGuardians.groovy

```diff
@@ -779,6 +786,14 @@ public enum CrystalGuardians implements LogicCardInfo {
       case IGGLYBUFF_21:
       return basic (this, hp:HP050, type:C, retreatCost:1) {
         weakness F
+        pokeBody "Hover Lift", {
+          text "You pay [C] less to retreat your Jigglypuff, Wigglytuff, Wigglytuff ex, and Igglybuff."
+          getterA GET_RETREAT_COST ,{ h->
+            if (h.effect.target.owner == self.owner && ["Jigglypuff", "Wigglytuff", "Wigglytuff ex", "Igglybuff"].contains(h.effect.target.name)) {
+              h.object -= 1
+            }
+          }
+        }
         pokePower "Baby Evolution", {
```

Comment:
Just making a note here that this Poké-Power isn't on the carddb and should probably be added at some point.